$OpenBSD: patch-configure,v 1.1.1.1 2001/09/07 05:44:32 pvalchev Exp $

- make configure non-interactive
- use system pthreads and nothing else

--- configure.orig	Sat Aug 11 00:22:30 2001
+++ configure	Wed Sep  5 11:04:16 2001
@@ -552,11 +552,6 @@ To contact one of the authors, send elec
 or paper mail to: Gnut, c/o Robert Munafo, P.O. Box 0513, Scituate, MA
 02066-0513, USA.
 
-Hit return:
-cat_eof
-read GPL_OK_TMP
-
-cat << 'cat_eof'
   - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 Read this only if you're interested in submitting patches or
 suggestions for improvements:
@@ -570,9 +565,7 @@ suggestions for improvements:
     change files and rebuild.  Please read that file before
     submitting patches to the makefiles or scripts.
 
-Hit return:
 cat_eof
-read GPL_OK_TMP
 
 ac_aux_dir=
 for ac_dir in $srcdir $srcdir/.. $srcdir/../..; do
@@ -2477,7 +2470,7 @@ else
 fi
 
 
-g_link_threads=no
+g_link_threads=noway
 
 if test "$g_link_threads" = no ; then
 		echo $ac_n "checking for pthread_create in -lpthread""... $ac_c" 1>&6
@@ -2527,6 +2520,8 @@ fi
 		g_link_threads=posix
 	fi
 fi
+
+g_link_threads=no
 
 if test "$g_link_threads" = no ; then
 		echo $ac_n "checking for pthread_create with -pthreads""... $ac_c" 1>&6
